spec/core/v2/ics-024-host-requirements/README.md

```diff
@@ -103,30 +103,61 @@ The `privateStore`:
 >
 > Note: this interface does not necessitate any particular storage backend or backend data layout. State machines may elect to use a storage backend configured in accordance with their needs, as long as the store on top fulfils the specified interface and provides commitment proofs.
 
-### Path-space
+### Provable Path-space
 
-At present, IBC/TAO recommends the following path prefixes for the `provableStore` and `privateStore`.
+IBC/TAO implementations MUST implement the following paths for the `provableStore` in the exact format specified. This is because counterparty IBC/TAO implementations will construct the paths according to this specification and send it to the light client to verify the IBC specified value stored under the IBC specified path.
 
 Future paths may be used in future versions of the protocol, so the entire key-space in the provable store MUST be reserved for the IBC handler.
 
-Keys used in the provable store MAY safely vary on a per-client-type basis as long as there exists a bipartite mapping between the key formats
-defined herein and the ones actually used in the machine's implementation.
 
-Parts of the private store MAY safely be used for other purposes as long as the IBC handler has exclusive access to the specific keys required.
-Keys used in the private store MAY safely vary as long as there exists a bipartite mapping between the key formats defined herein and the ones
-actually used in the private store implementation.
+| Store          | Path format                                                                | Value type        | Defined in |
+| -------------- | -------------------------------------------------------------------------- | ----------------- | ---------------------- |
+| provableStore  | "commitments/channels/{identifier}/sequences/{bigEndianUint64Sequence}"    | bytes             | [ICS 4](../ics-004-packet-semantics) |
```

Comment:
I'm wondering if there is some space for commitment collision which could led to double usage, but I cannot really think of a scenario that would allow that. At a first glance the provableStore paths look good to me.